cvat-core/src/session.js

```diff
@@ -1549,6 +1550,16 @@
                     cloudStorageId: {
                         get: () => data.cloud_storage_id,
                     },
+                    sortingMethod: {
+                        /**
+                         * @name enabled
+                         * @type {string}
+                         * @memberof module:API.cvat.enums.SortingMethod
```

Comment:
correct memberof is ``module:API.cvat.classes.Task``
``module:API.cvat.enums.SortingMethod`` is probably a type

Reply:
Sorry, fixed.